geode-for-redis/src/test/java/org/apache/geode/redis/internal/data/delta/SetTimestampDeltaUnitTest.java

```diff
@@ -23,18 +23,19 @@
 import java.io.DataOutputStream;
 
 import org.junit.Test;
+import org.junit.runner.RunWith;
 
-import org.apache.geode.redis.internal.data.delta.ReplaceByteArrayAtOffset;
-
-public class DeltaClassesJUnitTest {
+import org.apache.geode.redis.internal.data.RedisString;
+import org.apache.geode.test.junit.runners.GeodeParamsRunner;
 
+@RunWith(GeodeParamsRunner.class)
+public class SetTimestampDeltaUnitTest {
   @Test
-  public void testReplaceByteArrayAtOffsetForRedisString() throws Exception {
+  public void testSetTimestampDelta() throws Exception {
```

Comment:
Since it should be possible to set the timestamp on any class that extends `AbstractRedisData`, we should probably have tests for each of them. It's a bit tedious and duplicates some test code, but better safe than sorry.

Reply:
Since the actual code is in AbstractRedisData, which they all inherit from, I didn't think it necessary. Are we worried about someone overriding it?

Reply:
Yeah, it's a public, non-final method, so it's entirely possible that it could get overridden, even if there's no reason to right now. One alternative solution could be to make the method `final` so that we know it can't be overridden.

Reply:
Making it final seems like the right call. Done.